pkg/connector/tickets.go

```diff
@@ -162,7 +162,7 @@ func (j *Jira) schemaForProjectIssueType(ctx context.Context, project *jira.Proj
 
 	issueTypeCustomFields, err := j.getCustomFieldsForIssueType(ctx, project.ID, issueType)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("error getting custom fields for issue type %s: %w", issueType.ID, err)
```

Comment:
I would also include project id in the error here

Reply:
Thank you, added in [3964af8](https://github.com/ConductorOne/baton-jira/pull/25/commits/3964af867b05a9947e10e8f5e080d357b8d01964)